### PR TITLE
sql: 1. add doc for date/time functions 2. add doc for operators

### PR DESCRIPTION
**Pull request by XuHuaiyu**
PTAL @andelf @queenypingcap

### CODE REVIEW

File: sql/date-and-time-functions.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+title: TiDB 用户文档
+category: user guide
+---
+
+## 日期和时间函数
```
Comment (andelf) [inline sql/date-and-time-functions.md]: better use level 1 heading

File: sql/operators.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+category: user guide
+---
+
+
+
+# 操作符
+
+
+
+1. [操作符优先级](#操作符优先级)
```
Comment (andelf) [inline sql/operators.md]: A TOC?
missing heading for `[操作符优先级](#操作符优先级)`

File: sql/operators.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+2. [比较方法和操作符](#比较方法和操作符)
+3. [逻辑操作符](#逻辑操作符)
+4. [赋值操作符](#赋值操作符)
+
+| 操作符名                                     | 功能描述                                     |
+| ---------------------------------------- | ---------------------------------------- |
```
Comment (andelf) [inline sql/operators.md]: column width ratio is not good for this type of table

File: sql/operators.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+| [`:=`](https://dev.mysql.com/doc/refman/5.7/en/assignment-operators.html#operator_assign-value) | 赋值                                       |
+| [`BETWEEN ... AND ...`](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html#operator_between) | 判断值满足范围                                  |
+| [`BINARY`](https://dev.mysql.com/doc/refman/5.7/en/cast-functions.html#operator_binary) | 将一个字符串转换为一个二进制字符串                        |
+| [`&`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-and) | 位与                                       |
+| [`~`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-invert) | 位非                                       |
+| [`\|`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-or) | 位或                                       |
+| [`^`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_bitwise-xor) | 位异或                                      |
```
Comment (andelf) [inline sql/operators.md]: `按位异或` is more recognized

File: sql/operators.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+OR, ||
+= (assignment), :=
+```
+
+详情参见 [这里](https://dev.mysql.com/doc/refman/5.7/en/operator-precedence.html).
+
+## 比较方法和操作符
+
+| 操作符名                                     | 功能描述              |
+| ---------------------------------------- | ----------------- |
```
Comment (andelf) [inline sql/operators.md]: column width should reflect width of its body not heading.

**Comment by XuHuaiyu** [comment]
PTAL @andelf

File: sql/operators.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
...
+| [`\|\|`, `OR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_or) | 逻辑或                                      |
+| [`+`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_plus) | 加                                        |
+| [`REGEXP`](https://dev.mysql.com/doc/refman/5.7/en/regexp.html#operator_regexp) | 使用正则表达式进行模式匹配                            |
+| [`>>`](https://dev.mysql.com/doc/refman/5.7/en/bit-functions.html#operator_right-shift) | 右移                                       |
+| [`RLIKE`](https://dev.mysql.com/doc/refman/5.7/en/regexp.html#operator_regexp) | REGEXP 同义词                               |
+| [`*`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_times) | 乘                                        |
+| [`-`](https://dev.mysql.com/doc/refman/5.7/en/arithmetic-functions.html#operator_unary-minus) | 取反符号                                     |
+| [`XOR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_xor) | 逻辑亦或                                     |
```
Comment (andelf) [inline sql/operators.md]: typo

File: sql/operators.md [hunk 7]
```diff
@@ -0,0 +1,131 @@
...
+详情参见 [这里](https://dev.mysql.com/doc/refman/5.7/en/comparison-operators.html).
+
+## 逻辑操作符
+
+| 操作符名                                     | 功能描述 |
+| ---------------------------------------- | ---- |
+| [`AND`, `&&`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_and) | 逻辑与  |
+| [`NOT`, `!`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_not) | 逻辑非  |
+| [`\|\|`, `OR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_or) | 逻辑或  |
+| [`XOR`](https://dev.mysql.com/doc/refman/5.7/en/logical-operators.html#operator_xor) | 逻辑亦或 |
```
Comment (andelf) [inline sql/operators.md]: typo

**Pull request merged**